debugtools/DDR_VM/src/com/ibm/j9ddr/corereaders/tdump/TDumpReader.java

```diff
@@ -411,30 +418,58 @@ public int getBytesAt(long address, byte[] buffer, int offset,
 				int length) throws MemoryFault
 		{
 			try {
-				addressSpace.read(address, buffer, offset, length);
+				if (PatchedMemoryList.isPatched(address, length) == false) {
+					addressSpace.read(address, buffer, offset, length);
+					return length;
+				}
+				List<MemoryRecord> list = PatchedMemoryList.getPatched(address, length);
+				if (list == null) {
+					return getBytesAt(address, buffer, 0, buffer.length);
```

Comment:
This looks like it will yield infinite recursion, besides ignoring the supplied `offset` and `length` parameters.